cmd/tetragon/main.go

```diff
@@ -382,6 +390,77 @@ func tetragonExecute() error {
 	return obs.Start(ctx)
 }
 
+func loadTpFromDir(ctx context.Context, dir string) error {
+	tpMaxDepth := 1
+	tpFS := os.DirFS(dir)
+
+	if dir == defaults.DefaultTpDir {
+		// If the default directory does not exist then do not fail
+		// Probably tetragon not fully installed, developers testing, etc
+		if _, err := os.Stat(dir); os.IsNotExist(err) {
+			log.WithField("tracing-policy-dir", dir).Info("Loading Tracing Policies from directory ignored, directory does not exist")
+			return nil
+		}
+	}
+
+	err := fs.WalkDir(tpFS, ".", func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if d.IsDir() {
+			if strings.Count(path, string(os.PathSeparator)) >= tpMaxDepth {
```

Comment:
This could be common defensive practice to avoid walking nested trees , especially since this could be set as a configuration option that may point anywhere... recursing all directories without a limit could take time,  people may have volumes with symlinks, etc and we don't chase symlinks , so just making it 1 directory is simple for now... we can improve later if necessary.